### ISSUE TITLE
Upload and attach images to posts and pages

### ISSUE THREAD

**Issue by mariocasciaro** [hadronjs/hadron#5]

**Comment by jasonmarah** [comment]
Whoops, never noticed this issue. PR https://github.com/hadronjs/hadron/pull/9 sort of addresses this, or at least is a start.

**Comment by mariocasciaro** [comment]
Yes, this requires the creation of a more generic component to be shared across several contexts (post attachments, headline, avatar, etc...)
